### PR TITLE
Issue-260: RHEL 8.6 support

### LINKED ISSUE #260: Documentation📄: Add RHEL 8.6 support to Docs
### What is the current documentation state?

Add RHEL 8.6 (and other RHEL 8.4, 8.5) as a supported edge device (x86_64)
to `open-horizon.github.io/docs/installing/adding_devices.md`

Discuss if Open Horizon will also support RHEL 8.6 (and other RHEL 8.4, 8.5) as a supported edge device (ppc64le)

### PR DESCRIPTION
**Pull request by johnwalicki**
Signed-off-by: John Walicki <johnwalicki@gmail.com>

## Description

Fixes #260  and lots of linter warnings

This is a sub-task of https://github.com/open-horizon/anax/issues/3228

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

...

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/installing/adding_devices.md [hunk 1]
```diff
@@ -64,29 +65,29 @@ The following instructions guide you through the process of installing the requi
...
+- (optional) Sensor hardware: Many {{site.data.keyword.horizon}} edge services require specialized sensor hardware.
 
 ### Procedure
 {: #proc-x86}
 
-Prepare your device by installing a Debian, {{site.data.keyword.rhel}} 7.x, or Ubuntu {{site.data.keyword.linux_notm}}. The instructions in this content are based on a device that uses Ubuntu 18.x.
+Prepare your device by installing a Debian, {{site.data.keyword.rhel}}, or Ubuntu {{site.data.keyword.linux_notm}}. The instructions in this content are based on a device that uses Ubuntu 18.x.
 
-Install the most recent version of Docker on your device. For more information, see [Install Docker](https://docs.docker.com/engine/install/ubuntu/).
+Install the most recent version of Docker on your device. For more information, see [Install Docker ](https://docs.docker.com/engine/install/ubuntu/){:target="_blank"}{: .externalLink}.
```
Comment (joewxboy) [inline docs/installing/adding_devices.md:90]: Do we need to add a caveat here about RHEL 8.x and 9.x if they have Rodman 4.1 installed?  I assume these specific versions don't need Docker.

File: docs/installing/adding_devices.md [hunk 2]
```diff
@@ -269,11 +280,12 @@ Now that your edge device is prepared, continue on to [Installing the agent](reg
 ### Hardware requirements
 {: #hard-req-mac}
 
-* 2010 or later 64-bit {{site.data.keyword.intel}} Mac device
-* MMU virtualization is required.
-* MacOS X version 10.11 ("El Capitan") or later
-* An internet connection for your machine (wired or wifi)
-* (optional) Sensor hardware: Many {{site.data.keyword.horizon}} edge services require specialized sensor hardware.
+- 2010 or later 64-bit {{site.data.keyword.intel}} Mac device
```
Comment (joewxboy) [inline docs/installing/adding_devices.md:283]: Do we need to specify Intel-only Macs right now, and not arm-based Macs?

File: docs/installing/adding_devices.md [hunk 3]
```diff
@@ -24,35 +24,36 @@ The following instructions guide you through the process of installing the requi
...
+  - SuSE 15 SP2
+- ppc64le
+  - {{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines that run Ubuntu 20.x (focal) or Ubuntu 18.x (bionic)
+  - {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, and 8.3
+- ARM (32-bit)
+  - {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspberry Pi OS buster or bullseye
+- ARM (64-bit)
+  - {{site.data.keyword.linux_notm}} on ARM (64-bit), for example NVIDIA Jetson Nano, TX1, or TX2, running Ubuntu 18.x (bionic)
+- Mac
+  - {{site.data.keyword.macOS_notm}}
```
Comment (joewxboy) [inline docs/installing/adding_devices.md:41]: Do we need to specify Intel-only Macs, not arm-based Macs?

**Review by joewxboy — CHANGES_REQUESTED** [review]
Optional comments inline

File: docs/installing/adding_devices.md (hunk 1 above)
Comment (johnwalicki) [reply docs/installing/adding_devices.md:90]: Good point.  Fixed. Podman is also available on Ubuntu.  Docker is now optional.

File: docs/installing/adding_devices.md (hunk 3 above)
Comment (johnwalicki) [reply docs/installing/adding_devices.md:41]: I took a separate todo yesterday to open a ticket about doc changes to clarify that Open Horizon supports Intel-only Macs, not arm-based Macs.  This is a RHEL ticket so I won't fix that here.  Defer to #262

File: docs/installing/adding_devices.md (hunk 2 above)
Comment (johnwalicki) [reply docs/installing/adding_devices.md:283]: I took a separate todo yesterday to open a ticket about doc changes to clarify that Open Horizon supports Intel-only Macs, not arm-based Macs. This is a RHEL ticket so I won't fix that here. Defer to https://github.com/open-horizon/open-horizon.github.io/issues/262

**Comment by johnwalicki** [comment]
@joewxboy This ready to merge.  Ben has re-built the anax / agent-install scripts.

**Review by joewxboy — APPROVED** [review]
I don't know if "podman" is capitalized or not, but that's trivial.  Should it be externalized into a variable so that it doesn't get translated (eventually)?

**Comment by johnwalicki** [comment]
Good question about podman vs Podman.  The https://podman.io/ seems to reference it as `podman` the command, and `Podman` the project.  Open Horizon doesn't provide translations so low priority to move it to a variable.  Will include that in an upcoming Fedora 36 new Linux distro request. (I'll add Fedora as a variable too)

**Pull request merged**